qemu/tests/cfg/fio_windows.cfg

```diff
@@ -18,10 +18,10 @@
             image_aio = threads
     i386, i686:
         install_path = "C:\Program Files\fio"
-        install_cmd = 'msiexec /a DRIVE:\fio-x86.msi /qn TARGETDIR="${install_path}"'
+        install_cmd = 'msiexec /a DRIVE:\fio-3.1-x86.msi /qn TARGETDIR="${install_path}"'
     x86_64:
         install_path = "C:\Program Files (x86)\fio"
-        install_cmd = 'msiexec /a DRIVE:\fio-x64.msi /qn TARGETDIR="${install_path}"'
+        install_cmd = 'msiexec /a DRIVE:\fio-3.1-x64.msi /qn TARGETDIR="${install_path}"'
```

Comment:
I think we can leave only install_path platform-specific and move install_cmd out.